Source/Protobuf/Extensions.cs

```diff
@@ -16,113 +18,94 @@ namespace Dolittle.Protobuf
         /// </summary>
-        /// <param name="executionContext"><see cref="Execution.ExecutionContext"/> to convert from.</param>
-        /// <returns>Converted <see cref="ByteString"/>.</returns>
-        public static ByteString ToByteString(this Execution.ExecutionContext executionContext)
-        {
-            return executionContext.ToProtobuf().ToByteString();
-        }
+        /// <param name="failureReason">The reason for failure.</param>
+        /// <param name="failureId">The failure id.</param>
+        /// <returns>Converted <see cref="grpc.Failure" />.</returns>
+        public static grpc.Failure ToProtobufFailure(this ConceptAs<string> failureReason, Guid failureId) => failureReason.Value.ToProtobufFailure(failureId);
 
         /// <summary>
-        /// Convert from <see cref="Claims"/> to <see cref="IEnumerable{T}"/> of <see cref="Security.Contracts.Claim"/>.
+        /// Convert a <see cref="string" /> to <see cref="grpc.Failure" /> with not set <see cref="Guid" /> as failure id.
         /// </summary>
-        /// <param name="claims"><see cref="Claims"/> to convert from.</param>
-        /// <returns><see cref="IEnumerable{T}"/> of <see cref="Security.Contracts.Claim"/>.</returns>
-        public static IEnumerable<Security.Contracts.Claim> ToProtobuf(this Claims claims)
-        {
-            return claims.Select(_ => new Security.Contracts.Claim
-            {
-                Key = _.Name,
-                Value = _.Value,
-                ValueType = _.ValueType
-            });
-        }
+        /// <param name="failureReason">The reason for failure.</param>
+        /// <returns>Converted <see cref="grpc.Failure" />.</returns>
+        public static grpc.Failure ToProtobufFailure(this string failureReason) => failureReason.ToProtobufFailure(Guid.Empty);
 
         /// <summary>
-        /// Convert from <see cref="IEnumerable{T}"/> of <see cref="Security.Contracts.Claim"/> to <see cref="Claims"/>.
+        /// Convert a <see cref="string" /> to <see cref="grpc.Failure" /> with given <see cref="Guid" /> as failure id.
         /// </summary>
-        /// <param name="claims"><see cref="IEnumerable{T}"/> of <see cref="Security.Contracts.Claim"/> to convert from.</param>
-        /// <returns>Converted <see cref="Claims"/>.</returns>
-        public static Claims ToClaims(this IEnumerable<Security.Contracts.Claim> claims)
-        {
-            return new Claims(claims.Select(_ => new Claim(_.Key, _.Value, _.ValueType)));
-        }
+        /// <param name="failureReason">The reason for failure.</param>
+        /// <param name="failureId">The failure id.</param>
+        /// <returns>Converted <see cref="grpc.Failure" />.</returns>
+        public static grpc.Failure ToProtobufFailure(this string failureReason, Guid failureId) =>
+            new grpc.Failure { Reason = failureReason, Id = failureId.ToProtobuf() };
```

Comment:
I don't think we should have these - we want to always construct from the `Failure`type.

I can't add a suggestion to remove, but my suggestion is just to get rid of them.

Reply:
Yes, they were removed in a later PR. But I removed them here :)